crates/js-sys/tests/wasm/JSON.rs

```diff
@@ -63,7 +70,22 @@ fn stringify() {
     arr.push(&JsValue::from(true));
     arr.push(&JsValue::from("hello"));
 
-    let str = JSON::stringify(&JsValue::from(arr));
+    let str = JSON::stringify(&JsValue::from(arr)).unwrap();
     let rust_str: String = From::from(str);
     assert_eq!(rust_str, "[1,true,\"hello\"]");
-}
\ No newline at end of file
+}
+
+#[wasm_bindgen_test]
+fn stringify_error() {
+    let func = Function::new_no_args("throw new Error(\"rust really rocks\")");
+    let obj = Object::new();
+    set_in_object(&obj, "toJSON", &JsValue::from(func));
```

Comment:
This should do it:

```rust
let prop = JsValue::from("toJSON");
Reflect::set(obj.as_ref(), prop.as_ref(), func.as_ref());
```

Reply:
Great, thanks!  Pushed 1 more commit that fixes this